integrations/common/vars.go

```diff
@@ -38,6 +38,11 @@ func EncodeOAuthData(t *oauth2.Token) OAuthData {
 	}
 }
 
+// IsOAuthTokenEmpty returns true if the OAuth 2.0 token is empty.
+func IsOAuthTokenEmpty(vs sdktypes.Vars) bool {
+	return vs.GetValue(OAuthAccessTokenVar) == ""
```

Comment:
Are there any other limitation we can check for validity? I assume `"x"` is not a valid token.

Reply:
Unfortunately there are no rules about the content of an OAuth access token - each service provider has their own conventions.